### PR TITLE
Add button to deploy to Heroku

### LINKED ISSUE #1: Still uses "hunter2" as admin password
Even after changing the admin password in config.json, the application still requires "hunter2" as the password for the admin account.

### PR DESCRIPTION
**Pull request by anishathalye**
Addresses #25 

Okay, so I just went ahead and did it. I think a Deploy to Heroku button is nice to have cause it let's people get started really quickly (it's basically zero configuration / setup effort required).

This patch also fixes #1

### CODE REVIEW

**Comment by anishathalye** [comment]
Bump. Do you think this is useful, or is it too much code complexity?

File: server/startup.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+    // file doesn't exist, but that's okay
+  }
+  if (configJson) {
+    config = JSON.parse(configJson);
+  }
+
+  // environment variables override config file
+  var configTemplate = JSON.parse(Assets.getText('config.json.template'));
+  var envConfigs = readConfigsFromEnv(configTemplate);
+  overlay(config, envConfigs);
```
Comment (ehzhang) [inline server/startup.js:18]: Why not just use `Object.assign` instead of a custom `overlay` function?

**Review by ehzhang — COMMENTED** [review]
Sorry for the slow review - this looks generally good, save for the one bit about `overlay`.

Thanks for thinking about backwards compatibility :)

Though I think if I have the time I should update all of the libraries and eventually get it using `Meteor.settings`...

**Comment by anishathalye** [comment]
We need something like `overlay` because `Object.assign` doesn't do a deep copy.

**Review by ehzhang — APPROVED** [review]
SGTM

**Pull request merged**